tests/js/environment.js

```diff
@@ -1,23 +1,28 @@
 var fluid = require("infusion");
 
-fluid.defaults("gpii.pouch.tests.environment", {
+fluid.defaults("gpii.test.pouch.environment", {
     gradeNames: ["fluid.test.testEnvironment"],
     port:       6792,
     baseUrl:    "http://localhost:6792/",
     testUrl:    "/sample/",
```

Comment:
The `testUrl` was cruft and has been removed.  I also upgrade `baseUrl` to use `fluid.stringTemplate` and the `port` option, and tidied up the usages throughout the package.